### PR TITLE
Openwebicons

### PR DESCRIPTION
**Pull request by pfefferle**
Added OpenWebIcons font: http://pfefferle.github.io/openwebicons/

### CODE REVIEW

**Comment by jimaek** [comment]
Hey, sorry it took so long.
Ignore the bot and add `./` in front of your files to retain the directory names

File: files/openwebicons/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "openwebicons",
+  "repo": "pfefferle/openwebicons",
+  "files": {
+    "basePath": "",
```
Comment (jimaek) [inline files/openwebicons/update.json]: You can remove this completely.

**Comment by pfefferle** [comment]
No problem :)

I re-added the `./` and removed the `basePath`.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**